src/app/work-item/work-item-list/work-item-detail/work-item-link/work-item-link.component.ts

```diff
@@ -90,98 +85,54 @@ export class WorkItemLinkComponent implements OnInit, OnChanges {
         
     }
 
-    onSelectRelation(relation: LinkType): void{
+    onSelectRelation(relation: any): void{
```

Comment:
The data we are getting in this param is not an entire LinkType any more, it's some manipulated data like just the link type ID, name (forward or reverse name).